go.mod

```diff
@@ -281,3 +281,5 @@ require (
 	mvdan.cc/unparam v0.0.0-20211002134041-24922b6997ca // indirect
 	software.sslmate.com/src/go-pkcs12 v0.0.0-20210415151418-c5206de65a78 // indirect
 )
+
+replace github.com/u-root/u-root v0.9.0 => github.com/patmaddox/u-root v0.0.0-20221101021341-da6a6f63938d
```

Comment:
Of course this is just for purposes of this PR. `u-root` will need to be updated.